src/ServiceControl.UnitTests/Archiving/ArchiveGroupTests.cs

```diff
@@ -65,5 +65,60 @@ await handler.Handle(message, context)
                 }
             }
         }
+
+        [Test]
+        public async Task ArchiveGroup_GetGroupDetails_doesnt_fail_with_invalid_groupId()
+        {
+            // Arrange
+            using (var documentStore = InMemoryStoreBuilder.GetInMemoryStore())
+            {
+                var failureGroupsViewIndex = new FailureGroupsViewIndex();
+                await failureGroupsViewIndex.ExecuteAsync(documentStore);
+
+                var groupId = "TestGroup";
+                var previousArchiveBatchId = ArchiveBatch.MakeId(groupId, ArchiveType.FailureGroup, 1);
+
+                using (var session = documentStore.OpenAsyncSession())
+                {
+                    var previousAchiveBatch = new ArchiveBatch { Id = previousArchiveBatchId };
```

Comment:
Spelling in `previousAchiveBatch `

Reply:
Thanks, also fixed spelling in other test